### PR TITLE
PAYEE-760 - Rename method to allow for incremental approach to refactor

### PR DESCRIPTION
**Pull request by brendanstraughan**
- [ ] Unit tests passing
- [ ] Coverage reached
- [ ] Acceptance tests passing
- [ ] Reviewed

### CODE REVIEW

**Review by Jake-Raffe — APPROVED** [review]
lgtm

File: app/uk/gov/hmrc/tai/service/TaxAccountService.scala [hunk 1]
```diff
@@ -49,7 +50,10 @@ class TaxAccountService @Inject()(taxAccountConnector: TaxAccountConnector) {
     implicit hc: HeaderCarrier): Future[Either[String, Option[TaxCodeIncome]]] =
     EitherT(taxAccountConnector.taxCodeIncomes(nino, year)).map(_.find(_.employmentId.contains(employmentId))).value
 
-  def taxAccountSummary(nino: Nino, year: TaxYear)(implicit hc: HeaderCarrier): Future[TaiResponse] =
+  def taxAccountSummaryOld(nino: Nino, year: TaxYear)(implicit hc: HeaderCarrier): Future[TaiResponse] =
```
Comment (PerrettJ4) [inline app/uk/gov/hmrc/tai/service/TaxAccountService.scala:53]: Why do we have this old method?

**Review by PerrettJ4 — APPROVED** [review]
👍

File: app/uk/gov/hmrc/tai/service/TaxAccountService.scala (hunk 1 above)
Comment (brendanstraughan) [reply app/uk/gov/hmrc/tai/service/TaxAccountService.scala]: we shouldnt, good spot, I have removed them now

**Review by sergi-hmrc — APPROVED** [review]

**Pull request merged**